README.md

```diff
@@ -48,7 +48,7 @@ docker-compose up
 11. Still in the new CMD or terminal, hit the API with a request
 
 ```
-curl localhost:8000/v1/images?q=honey
+curl 'http://localhost:8000/v1/images/?format=json&q=honey'
```

Comment:
It works without `http://` and `format=json`, but you do need the quotes, at least for my installation of zsh. Otherwise, the question mark causes it to throw an error.

Reply:
In my environment where I use bash,  only "curl localhost:8000/v1/images?q=honey" doesn't work.

Reply:
Does`curl "localhost:8000/v1/images?q=honey"`work? With quotation marks around the url.

Reply:
The problem is just the trailing slash after `images`, even this would have worked:
```suggestion
curl localhost:8000/v1/images/?q=honey
```